merkle/coniks/coniks.go

```diff
@@ -107,27 +107,27 @@ func (m *Hasher) BitLen() int {
 	return m.Size() * 8
 }
 
-// writeMaskedNodeID writes the node ID bits to the buffer, padded with zero
-// bits to the byte Size() of the hashes in use by this hasher.
+// writeMaskedID writes the node ID bits to the buffer, padded with zero bits
+// to the byte Size() of the hashes in use by this hasher.
 //
 // TODO(pavelkalinnikov): We must not use BitLen() and Size() interchangeably.
 // The tree height and hash size could be different.
 // TODO(pavelkalinnikov): Padding with zeroes doesn't buy us anything, as the
 // depth is also written to the Buffer.
-func (m *Hasher) writeMaskedNodeID(b *bytes.Buffer, node tree.NodeID2) {
```

Comment:
opinion: I preferred the previous method name. In normal conversation one would still refered to `node.ID` as a "node ID".

Reply:
Fair point, restored the original name.